dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/ServiceNameServlet.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.servlet.annotation.WebServlet;
+
+@WebServlet("/ServiceNameServlet")
+@ApplicationScoped
+public class ServiceNameServlet extends FATServlet {
+
+    @Inject
+    Tracer tracer;
+
+    @Inject
+    Span span;
+
+    @Inject
+    OpenTelemetry openTelemetry;
+
+    //Tests if otel.service.name is set to the application name by default
+    @Test
+    public void testServiceNameConfig() {
+        Tracer tracer = openTelemetry.getTracer("config-test", "1.0.0");
+        Span span = tracer.spanBuilder("testSpan").startSpan();
+        span.end();
+        assertThat(openTelemetry.toString(), containsString("service.name=\"TelemetryApp\""));
```

Comment:
We should test the attributes on the span, not by doing `toString()` on the `openTelemetry` object. Use `InMemorySpanExporter`.